.zenodo.json

```diff
@@ -15,37 +15,37 @@
   "creators": [
     {
       "affiliation": "Department of Physics, TU Dortmund University, Otto-Hahn-Str. 4, 44221 Dortmund, Germany",
-      "name": "Maximilian N\u00f6the"
+      "name": "Maximilian N\u00f6the",
       "orcid": "0000-0001-7993-8189"
     },
     {
       "affiliation": "AIM, CEA, CNRS, Universite Paris-Saclay, Universite Paris Diderot, Sorbonne Paris Cite, F-91191 Gif-sur-Yvette, France",
-      "name": "Michele Peresano"
+      "name": "Michele Peresano",
       "orcid": "0000-0002-7537-7334"
     },
     {
       "affiliation": "Laboratoire d’Annecy de Physique des Particules, Univ. Grenoble Alpes, Univ. Savoie Mont Blanc, CNRS, LAPP, 74000 Annecy, France",
-      "name": "Thomas Vuillaume"
+      "name": "Thomas Vuillaume",
       "orcid": "0000-0002-5686-2078"
     },
     {
       "affiliation": "Department of Physics, TU Dortmund University, Otto-Hahn-Str. 4, 44221 Dortmund, Germany",
-      "name": "Lukas Nickel"
+      "name": "Lukas Nickel",
       "orcid": ""
     },
     {
       "affiliation": "Department of Physics, TU Dortmund University, Otto-Hahn-Str. 4, 44221 Dortmund, Germany",
-      "name": "Noah Biederbeck"
+      "name": "Noah Biederbeck",
       "orcid": ""
     },
     {
       "affiliation": "Institut de Fisica d'Altes Energies (IFAE), The Barcelona Institute of Science and Technology, Campus UAB, 08193 Bellaterra (Barcelona), Spain",
-      "name": "Lea Jouvin"
+      "name": "Lea Jouvin",
       "orcid": ""
```

Comment:
I would not give empty orcids, leave it out if unknown

Reply:
Done